src/main/java/uk/gov/hmcts/ccd/domain/model/callbacks/ItemType.java

```diff
@@ -0,0 +1,5 @@
+package uk.gov.hmcts.ccd.domain.model.callbacks;
+
+public enum ItemType {
```

Comment:
Could this be either renamed to `SignificantItemType` or moved to `SignificantItem.Type`?

Reply:
addressed